connexion/operation.py

```diff
@@ -76,7 +76,13 @@ def __init__(self, method, path, operation, app_produces, app_security,
         self.validate_responses = validate_responses
 
         self.operation = operation
-        self.operation_id = operation['operationId']
+        operation_id = operation['operationId']
+        router_controller = None
```

Comment:
You can reduce the amount of code by doing `router_controller = operation.get('x-swagger-router-controller')` as this will return None if `x-swagger-router-controller`is not defined.